handler_test.go

```diff
@@ -7,7 +7,9 @@ import (
 	"testing"
 	"time"
 
+	"fmt"
```

Comment:
Sorry about that. The IDE is integrated with go fmt, etc.. and usually orders correctly those. Not sure what happened.